secrets/secrets.go

```diff
@@ -68,6 +72,18 @@ func DecryptEnvelope(envelope *Envelope) ([]byte, error) {
 }
 
 func getKeyService(env, region, masterKeyID string) (KeyService, error) {
+	mu.Lock()
```

Comment:
heh thats smart, I hadn't thought of that as a workaround :P

Reply:
🥂